shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -78,6 +79,8 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       NamespaceNotFoundException {
     reader = shellState.getReader();
 
+    Boolean force = cl.hasOption(forceOpt);
```

Comment:
Could this be the primitive `boolean` instead of `Boolean`?

Reply:
I agree it should be:

```suggestion
    boolean force = cl.hasOption(forceOpt);
```